.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,42 @@
+name: Feature Request
+description: Suggest an idea for this project
+labels: [kind/enhancement]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature enhancement!
+  - type: textarea
+    id: describe-feature
+    attributes:
+      label: Describe the feature
+      description: Is your feature request related to a problem?  
+      placeholder: Please describe
+      value: "I'm always frustrated when ..."
+    validations:
+      required: true
+  - type: textarea
+    id: describe-solution
+    attributes:
+      label: Describe your solution
+      description: A clear and concise description of what you want to happen.
+      placeholder: What would a solution look like?
+      value: "It would be better if it did this ..."
+    validations:
+      required: true
+  - type: textarea
+    id: describe-alternatives
+    attributes:
+      label: Describe alternatives you've considered
+      description: A clear and concise description of any alternative solutions or features you've considered.
+      placeholder: What would a solution look like?
+      value: "I thought about doing it this way too ..."
```

Comment:
`"I thought about doing it this way too ..."` --> `"Some of the other solutions or features I have considered are ..."`